eslint-plugin/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "eslint-plugin-tsdoc",
+  "version": "0.1.0",
+  "description": "eslint plugin for tsdoc",
+  "private": false,
+  "license": "MIT",
+  "scripts": {
+    "build": "tsc",
+    "test": "node tests/index.js"
```

Comment:
This path should now be `lib/tests/index.js`.

Although it'd probably be best to include this in the build script so we run the test as part of the build.